Releases.md

```diff
@@ -19,6 +20,11 @@ See [DirectML version history on MSDN](https://docs.microsoft.com/windows/win32/
 | [1.1.0](#directml-110) | [DML_FEATURE_LEVEL_2_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_2_0) | Version 2004 (Build 10.0.19041; 20H1, "May 2020 Update") | -                                                                                         |
 | [1.0.0](#directml-100) | [DML_FEATURE_LEVEL_1_0](https://docs.microsoft.com/windows/win32/direct3d12/dml-feature-level-history#dml_feature_level_1_0) | Version 1903 (Build 10.0.18362; 19H1, "May 2019 Update") | -                                                                                         |
 
+# DirectML 1.9.1
+- UINT64 support for OneHot operator.
+- Improve FP32 convolution performance on NVIDIA graphics card.
+- Improve Concat performance on Intel DG2 and NVIDIA graphics cards.
+- Fix bugs in LSTM and Split node when executing within DML Graph.
```

Comment:
>LSTM [](http://example.com/codeflow?start=14&length=4)

That bug isn't actually about LSTM, despite the PR title. The LSTM just happened to split the DML graph partitions which revealed the real bug in Squeeze/Unsqueeze (which is the same issue as Squeeze/Unsqueeze in the refine model). DML doesn't have squeeze/unsqueeze of course, as they are just the elementwise identity operator. So really this is more about an identity elimination optimization gone wrong. So maybe

```
- Fix bug with unconnected split nodes when executing in DML graph.
- Fix identity node optimization when near end of DML graph.
```